__docs__/wonder-blocks-accordion/accordion-section.stories.tsx

```diff
@@ -0,0 +1,463 @@
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+import type {Meta, StoryObj} from "@storybook/react";
+
+import {AccordionSection} from "@khanacademy/wonder-blocks-accordion";
+import Button from "@khanacademy/wonder-blocks-button";
+import {DetailCell} from "@khanacademy/wonder-blocks-cell";
+import {View} from "@khanacademy/wonder-blocks-core";
+import Icon, {icons} from "@khanacademy/wonder-blocks-icon";
+import {Strut} from "@khanacademy/wonder-blocks-layout";
+import {tokens} from "@khanacademy/wonder-blocks-theming";
+import {LabelLarge} from "@khanacademy/wonder-blocks-typography";
+
+import ComponentInfo from "../../.storybook/components/component-info";
+import packageConfig from "../../packages/wonder-blocks-icon/package.json";
+
+import AccordionSectionArgtypes from "./accordion-section.argtypes";
+
+export default {
+    title: "Accordion / AccordionSection",
+    component: AccordionSection,
+    parameters: {
+        componentSubtitle: (
+            <ComponentInfo
+                name={packageConfig.name}
+                version={packageConfig.version}
+            />
+        ),
+    },
+    argTypes: AccordionSectionArgtypes,
+} as Meta<typeof AccordionSection>;
+
+type StoryComponentType = StoryObj<typeof AccordionSection>;
+
+/**
+ * By default, an AccordionSection has a caret at the end of the header block
```

Comment:
**suggestion:** I think it would be good here to mention that the accordion section is uncontrolled by default and maybe linking to the uncontrolled story (?).